kefir.js.flow

```diff
@@ -0,0 +1,160 @@
+/* @flow */
+
+export type Event<T> =
+  {type: 'value', value: T} |
+  {type: 'error', value: any} |
+  {type: 'end', value: void};
+
+declare class _Emitter<T> {
+  value(value: T): boolean;
+  event(event: Event<T>): boolean;
+  error(e: any): boolean;
+  end(): void;
+
+  // Deprecated methods
+  emit(value: T): boolean;
+  emitEvent(event: Event<T>): boolean;
+}
+
+export type Emitter<T> = _Emitter<T>;
+
+export type Subscription = {
+  closed: boolean;
+  unsubscribe(): void;
+};
+
+export type Observer<T> = {
+  value(value: T): void;
+  error(err: any): void;
+  end(): void;
+};
+
+declare class Observable<+T> {
+  toProperty(): Property<T>;
+  toProperty<U>(getCurrent: ?() => U): Property<T|U>;
+  changes(): Observable<T>;
+
+  observe(obs: Observer<T>): Subscription;
+  observe(onValue: ?(i: T) => void, onError: ?(err: any) => void, onEnd: ?() => void): Subscription;
+  onValue(cb: (i: T) => void): Observable<T>;
```

Comment:
I never used this flow feature before, but I think `Observable<T>` can be replaced with `this` in this case.